class.jetpack.php

```diff
@@ -4629,7 +4629,9 @@ function build_connect_url( $raw = false, $redirect = false, $from = false ) {
 					'is_active'     => Jetpack::is_active(),
 					'jp_version'    => JETPACK__VERSION,
 					'auth_type'     => 'calypso',
-					'secret'		=> $secret,
+					'secret'        => $secret,
```

Comment:
Fixed some indentation issues (tabs instead of spaces) in 948a6aeae3c76a95e2e163fc5923c9f463b64fd0